source/pages/basics/basic_usage.rst

```diff
@@ -1,52 +1,59 @@
 .. _basic-usage:
 
-***********
-Basic Usage
-***********
+******************************
+Navigating the data repository
```

Comment:
Preferably when using the term 'data repository' use title capitals.

Reply:
So, in general refer to the 'service' or 'repository', but when using 'data repository' as a name use title capitals.